R/drive_cp.R

```diff
@@ -107,9 +111,7 @@ drive_cp <- function(file,
 
   out <- as_dribble(list(proc_res))
 
-  if (verbose) {
-    new_path <- paste0(append_slash(path$name), out$name)
-    message_glue("\nFile copied:\n  * {file$name} -> {new_path}")
-  }
+  new_path <- paste0(append_slash(path$name), out$name)
+  message_glue("\nFile copied:\n  * {file$name} -> {new_path}")
```

Comment:
FYI: `message_glue()` and friends will be replaced next, with cli-using functions. In a separate unit of work.